app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -98,16 +100,18 @@ private void nextActivity() {
      * @param v The view of the button being clicked.
      */
     public void buttonAction(final View v) {
-        final String button = v.getContentDescription().toString();
+        final int button = Integer.parseInt(v.getContentDescription().toString());
         LOGGER.info("button '%s' clicked", button);
```

Comment:
My bad 😅 could you change it tho, there is also a similar log on [line 116](https://github.com/ericcornelissen/NervousFish/pull/250/files/a121f5f95ec29150d5254fccab15f3a42bf7274c#diff-12207c280a7c5d460c92841b36106522R116)